lib/rubocop/cop/rails/avoid_duplicate_ignored_columns.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks for duplicate calls to ActiveRecord::Base.ignore_columns= as the
+      # subsequent calls will override the first call.
+      #
+      # @example
+      #   # bad
+      #   class User < ApplicationRecord
+      #     self.ignored_columns = %w(profile)
+      #     self.ignored_columns = %w(birthday)
+      #   end
+      #
+      #   # good
+      #   class User < ApplicationRecord
+      #     self.ignored_columns = %w(profile birthday)
+      #   end
+      #
+      class AvoidDuplicateIgnoredColumns < Base
+        MESSAGE = 'self.ignored_columns= has already been called on line %<other_line_number>s.'
```

Comment:
And can you use `RESTRICT_ON_SEND` constant?
```suggestion
        MESSAGE = 'self.ignored_columns= has already been called on line %<other_line_number>s.'
        RESTRICT_ON_SEND = %i[ignored_columns=].freeze
```